packages/govuk-frontend/src/govuk/components/accordion/accordion.yaml

```diff
@@ -48,30 +48,45 @@ params:
     required: true
     description: An array of sections within the accordion.
     params:
-      - name: heading.text
-        type: string
+      - name: heading
+        type: object
         required: true
-        description: The title of each section. If `heading.html` is supplied, this is ignored.
-      - name: heading.html
-        type: string
-        required: true
-        description: The HTML content of the header for each section. Used as the title for each section. The header is inside the HTML `<button>` element, so you can only add [phrasing content](https://html.spec.whatwg.org/#phrasing-content) to it.
-      - name: summary.text
-        type: string
-        required: false
-        description: Text content for summary line. If `summary.html` is supplied, this is ignored.
-      - name: summary.html
-        type: string
+        description: The heading of each section.
+        params:
+          - name: text
+            type: string
+            required: true
+            description: If `html` is set, this is not required. The heading text of each section. If `html` is provided, the `text` option will be ignored.
```

Comment:
I'm possibly missing some context, but the second line looks like it should come first. Also, what is the difference between the html being set as opposed to being provided?
Is it still accurate to put it this way round?
"The heading text of each section. If `html` is set, heading is not required.  If `html` is provided, the `text` option will be ignored."

Reply:
Yeah definitely, [take a look at all the other places we use this format](https://github.com/search?q=repo%3Aalphagov%2Fgovuk-frontend%20%22is%20set%2C%20this%20is%20not%20required%22&type=code) 😣 

Mind if we change them all in another pull request?